bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -119,6 +127,19 @@ public class FormContainerImpl extends AbstractContainerImpl implements FormCont
     @Default(values = DEFAULT_FORMS_SPEC_VERSION)
     private String specVersion;
 
+    @ValueMapValue(injectionStrategy = InjectionStrategy.OPTIONAL)
```

Comment:
Please use reserved properties for defining an property name.
Any new property name should be prefixed with fd namespace.

Reply:
Done